pkg/bootstrap/user/userprofile.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package user
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/trustbloc/edge-core/pkg/storage"
+)
+
+// Profile is the user's bootstrap profile.
+type Profile struct {
+	Sub               string
+	SDSPrimaryVaultID string
+	KeyStoreIDs       []string
+}
+
+// ProfileStore is the user Profile CRUD API.
+type ProfileStore struct {
+	s storage.Store
+}
+
+// NewStore returns a new ProfileStore.
+func NewStore(s storage.Store) *ProfileStore {
+	return &ProfileStore{s: s}
+}
+
+// Save saves the user Profile.
+func (ps *ProfileStore) Save(p *Profile) error {
+	bits, err := json.Marshal(p)
```

Comment:
I'm curious as to why you use the name `bits` instead of `bytes` (or something similar with "bytes" in the name). `bytes` to me seems like the intuitive name for a byte array... And so far from what I've seen, we use that naming convention in TrustBloc and Aries

Reply:
(this is also a nitpick)

Reply:
Would definitely not recommend `bytes` as it would clash with the standard golang `bytes` package.

Reply:
Good point.